src/Controller/SpaceController.php

```diff
@@ -11,15 +12,28 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
-#[Route('/space')]
 class SpaceController extends AbstractController
 {
-    #[Route('/', name: 'space_index', methods: ['GET'])]
-    public function index(SpaceRepository $spaceRepository): Response
+
+    /**
+     * @Route("/", name="app_home")
+     */
```

Comment:
keep the previous route declaration:

    #[Route('/', name: 'space_index', methods: ['GET'])]